src/SprykerEco/Zed/Braintree/Persistence/Propel/Schema/spy_braintree.schema.xml

```diff
@@ -58,13 +54,28 @@
         <foreign-key name="spy_braintree_transaction_status_log-fk_braintree" foreignTable="spy_payment_braintree">
             <reference foreign="id_payment_braintree" local="fk_payment_braintree"/>
         </foreign-key>
+        <foreign-key name="spy_braintree_transaction_status_log-fk_payment_braintree_order_item" foreignTable="spy_payment_braintree_order_item">
+            <reference foreign="id_payment_braintree_order_item" local="fk_payment_braintree_order_item"/>
+        </foreign-key>
         <behavior name="timestampable"/>
         <id-method-parameter value="spy_payment_braintree_transaction_status_log_pk_seq"/>
     </table>
 
+    <table name="spy_payment_braintree_transaction_order_item">
```

Comment:
This is a new table?

Reply:
Yes

Reply:
In that case, I don't really like naming. IMO it should describe that this is a connection table

Reply:
Renamed to spy_payment_braintree_transaction_status_log_to_order_item